velox/exec/Task.cpp

```diff
@@ -1422,6 +1475,7 @@ std::unique_ptr<ContinuePromise> Task::addSplitLocked(
   ++taskStats_.numTotalSplits;
   ++taskStats_.numQueuedSplits;
 
+  // Remove after presto switches to use the new API
```

Comment:
These comments would be hard to find later.
1. Do we need these comments? Would simply removing the old class members highlight these code sites with compilation errors, so no comment needed?
2. Could add some 'searchable key' to the comments, such as GH issue ID: `issueNNNNN`.